test/gradient/elixir_fmt_test.exs

```diff
@@ -18,6 +20,19 @@ defmodule Gradient.ElixirFmtTest do
     assert res == expected
   end
 
+  @tag :skip
+  describe "types format" do
+    test "wrong return type" do
+      {_tokens, ast} = load("/type/Elixir.WrongRet.beam", "/type/wrong_ret.ex")
+      opts = []
+      errors = type_check_file(ast, opts)
+
+      for e <- errors do
+        :io.put_chars(e)
```

Comment:
We don't assert anything in this test, do we? I think it's fine to have one or two tests that just use the expected literal binaries to assert if the formatting works as expected. Obviously, we won't do it for all possible error types, but a few are OK.

Reply:
Yes, this test case needs to assert something in the future. I used it only to display some results and it is set to be skipped. Should I try to provide some real tests in this PR or do it in a new one?

Reply:
That would be nice to have in this PR 👍

Reply:
I added tests for the cases defined in the `test/examples/type/wrong_ret.ex` 😃 I only check there if the error message contains the correct value. I don't want to compare the full message because the message could change in near future. Also, these checks should be replaced with something more precise in the future.